__tests__/components/editor/property/InputLookupQA.test.js

```diff
@@ -120,16 +120,6 @@ describe('<InputLookupQA />', () => {
       jest.restoreAllMocks()
     })
 
-    it('sets the default values according to the property template if they exist', () => {
```

Comment:
is there a test showing this from the new approach?  If it's not approp for a unit test, make a ticket to make it an integration test?

Reply:
This is already covered by https://github.com/LD4P/sinopia_editor/blob/010f086f6901b3e218d78c97e0f176e76ae75db5/__tests__/components/editor/property/InputLookupQA.test.js#L152-L160